src/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -62,6 +62,7 @@ public static IEnumerable<object[]> Parse_TestData()
             yield return new object[] { "1", false, UInt64Enum.One };
             yield return new object[] { "5", false, (UInt64Enum)5 };
 
+#if netcoreapp
```

Comment:
Are these #ifdef'ed because of missing APIs or because of differences in behavior? If it is API this is fine but we should consider splitting them into another file instead of a ton of #ifdef's. If it is behavior differences then we need to file bugs and get them investigated further as we shouldn't blindly disable large sets of tests for that.